tests/datasets/storages/processors/test_replaced_groups.py

```diff
@@ -103,7 +106,9 @@ def test_not_many_groups_to_exclude(query: ClickhouseQuery) -> None:
 
 def test_too_many_groups_to_exclude(query: ClickhouseQuery) -> None:
     state.set_config("max_group_ids_exclude", 2)
-    set_project_exclude_groups(2, [100, 101, 102], ReplacerState.EVENTS)
+    set_project_exclude_groups(
+        2, [100, 101, 102], ReplacerState.EVENTS, "replacement_type"
```

Comment:
Is this test testing your new changes without any new assertions? How would it fail if your code was incorrect?

Reply:
No, it's not testing my new changes but since I updated the signature for `set_project_exclude_group()`  I needed to update tests that used this function. It could be any string, this test isn't relevant to my changes. However, `tests/test_replacer.py` is updated to test my changes.